src/ngraph/runtime/cpu/pass/cpu_mkldnn_primitive_build.cpp

```diff
@@ -798,6 +1419,50 @@ bool MKLDNNPrimitiveBuildPass::run_on_call_graph(const std::list<std::shared_ptr
         }
     }
 
+    std::ofstream desc_file = std::ofstream(m_desc_filename, std::ios::out | std::ios::binary);
+    for (const auto& shp_node : nodes)
+    {
+        if (!in_new_map(shp_node))
+        {
+            continue;
+        }
+
+        Node* node = shp_node.get();
+
+        // skip in place concat, relu, reshape, slice.
```

Comment:
shouldn't in-place elimination happen in cpu_emitter? E.g., https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/runtime/cpu/cpu_emitter.cpp#L1404

Reply:
we cant also skip relu. you can only skip non-destructive in-place.

Reply:
If we don't skip here, we write descriptors to the desc_file but don't read from the file for these nodes. The node after such node would read the wrong descriptors and get segfault.